Components/Rating/rating.html

```diff
@@ -0,0 +1,254 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```

Comment:
Please include meta tag like author name & description

Reply:
Ok will do that after deadline! Have less time now

Reply:
Done